app/controllers/invite_schools_controller.rb

```diff
@@ -0,0 +1,20 @@
+class InviteSchoolsController < ApplicationController
```

Comment:
👍 but I think SchoolInvitesController might be a better name. It certainly makes more sense in the routes file where you declare the resource.